resources/helm-broker/charts/etcd-stateful/values.yaml

```diff
@@ -1,6 +1,6 @@
 tlsSetup:
```

Comment:
whole that section can be removed because I see that we are not using the tls job in Helm Broker